src/templates/template-text-classification/main.py

```diff
@@ -27,16 +27,9 @@ def run(local_rank: int, config: Any):
     manual_seed(config.seed + rank)
 
     # create output folder and copy config file to output dir
-    output_dir = config.output_dir
+    main_output_dir = config.output_dir
```

Comment:
Instead of these weird code can't we just save the config first and then define the output_dir ?